dotnet/src/InternalUtilities/src/System/DynamicUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel;
+
+internal static class DynamicUtils
+{
+    internal static T? TryGetPropertyValue<T>(object? obj, string propertyName, T? defaultValue = default)
```

Comment:
Suggest adding `OrDefault` in the end.

```suggestion
    internal static T? TryGetPropertyValueOrDefault<T>(object? obj, string propertyName, T? defaultValue = default)
```

Reply:
Class will be deleted